conda/meta.yaml

```diff
@@ -37,6 +37,7 @@ test:
     - moto
     - ts-dds
     - ts-idl
+    - salpy-test
```

Comment:
where is this package hosted? I though we agreed to move the package to anaconda.